decidim-core/app/views/decidim/shared/_orders.html.erb

```diff
@@ -1,9 +1,10 @@
 <div class="row collapse order-by">
   <div class="order-by__dropdown">
     <span class="order-by__text"><%= t("#{i18n_scope}.label") %></span>
-    <ul class="dropdown menu" data-dropdown-menu data-close-on-click-inside="false">
-      <li>
-        <%= order_link order, i18n_scope: i18n_scope %>
+    <ul class="dropdown menu" data-dropdown-menu>
```

Comment:
Thanks @mrcasals! I should consult git history more often instead of just removing things... :flushed:

I had a look at your link and the related issues, and it seems to have been fixed indeed in foundation 6.3.1 via https://github.com/zurb/foundation-sites/pull/9546, so this should be safe since we are already using that version.

It kind of bothers me that I've actually seen the behavior you describe in https://github.com/zurb/foundation-sites/issues/9778 during tests, and I thought https://github.com/decidim/decidim/pull/1407/commits/fbaa65451556c9e8f1986db8aa98584055d519c9 had fixed it. I guess, since there's a separate link under the dropdown, it can be mistakenly clicked for different unrelated reasons...

I think we should then fully revert #977, not just this menu. So maybe it's better that I remove this change and leave that for a separate PR?